### PR TITLE
Cl arrayofarray

### PR DESCRIPTION
**Pull request by chrislojacono**
## Description
Lots of stuff! got the homepage lookin decent and tons of js

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Review by Srfreeny — APPROVED** [review]

File: main.js [hunk 1]
```diff
@@ -530,45 +643,77 @@ const renderReviewForm = () => {
...
+    // LOOP THROUGH EACH ELEMENT.
+
+=======
     var ele = document.getElementsByTagName('input');
     let rating = document.getElementById('numberOfStars').selectedIndex + 1;
     let avatar;
+>>>>>>> master
     let product = document.getElementById("product").value;
     let name = document.getElementById("name").value;
+<<<<<<< HEAD
```
Comment (willkotheimer) [inline main.js:690]: This shows a merge conflict.

File: main.js [hunk 2]
```diff
@@ -530,45 +643,77 @@ const renderReviewForm = () => {
...
+    //If select comes out as 3 it is 4, if 0 it is 1
+    let rating = document.getElementById("numberOfStars").selectedIndex + 1;
+    let avatar;
+    // LOOP THROUGH EACH ELEMENT.
+
+=======
     var ele = document.getElementsByTagName('input');
     let rating = document.getElementById('numberOfStars').selectedIndex + 1;
     let avatar;
+>>>>>>> master
```
Comment (willkotheimer) [inline main.js:687]: This shows a merge conflict.

File: main.js [hunk 3]
```diff
@@ -530,45 +643,77 @@ const renderReviewForm = () => {
...
   if (buttonId === "submit") {
+<<<<<<< HEAD
+    var ele = document.getElementsByTagName("input");
+
+    //If select comes out as 3 it is 4, if 0 it is 1
+    let rating = document.getElementById("numberOfStars").selectedIndex + 1;
+    let avatar;
+    // LOOP THROUGH EACH ELEMENT.
+
+=======
```
Comment (willkotheimer) [inline main.js:683]: This shows a merge conflict.

File: main.js [hunk 4]
```diff
@@ -530,45 +643,77 @@ const renderReviewForm = () => {
...
+    console.log(name, product, rating, avatar, comment);
+=======
     for (let i = 0; i < ele.length; i++) {
       if (ele[i].type == 'radio' && ele[i].checked) {
         console.log(ele[i].checked);
         avatar = `${ele[i].id}.jpg`;
       }
     }
     let comment = document.getElementById("comment").value;
+>>>>>>> master
```
Comment (willkotheimer) [inline main.js:710]: This shows a merge conflict.

Comment (willkotheimer) [reply main.js:710]: Seeing lots of these lines means merge conflicts. Gitkraken is the easier way to solve these. Let me know if I can be of any help with that!

**Pull request merged**